generators/server/index.js

```diff
@@ -340,7 +340,7 @@ module.exports = JhipsterServerGenerator.extend({
                     message: function (response) {
                         return getNumberedQuestion('What is the folder path of your UAA application?.', currentQuestion, totalQuestions, function (current) {
                             currentQuestion = current;
-                        }, applicationType === 'gateway' && response.authenticationType === 'uaa');
+                        }, (applicationType === 'gateway' || applicationType === 'application') && response.authenticationType === 'uaa');
```

Comment:
we dont have an `applicationType ` called `application` is this a typo? also if you are changing the condition here then the same has to be updated for the when: block as well for the question